include/overlay005/bdhc.h

```diff
@@ -32,4 +35,15 @@ typedef struct BDHC {
     int stripsSize;
 } BDHC;
 
+BDHCUtil_CalculateObjectHeight(const fx32 objectHeight, const fx32 objectX, const fx32 objectZ, const BDHC *bdhc, fx32 *newObjectHeight);
```

Comment:
```suggestion
BOOL BDHC_CalculateObjectHeight(const fx32 objectHeight, const fx32 objectX, const fx32 objectZ, const BDHC *bdhc, fx32 *newObjectHeight);
```

Let's also remove the BDHCUtil prefix from the static functions in the C file, or at least change them to just `BDHC`.

Reply:
Normalized the prefixes to `BDHC` everywhere, looks way cleaner ^^